components/script/dom/url.rs

```diff
@@ -279,4 +279,9 @@ impl URLMethods for URL {
     fn SetUsername(&self, value: USVString) {
         UrlHelper::SetUsername(&mut self.url.borrow_mut(), value);
     }
+
+    // https://url.spec.whatwg.org/#dom-url-tojson
+    fn ToJSON(&self) -> USVString {
+        UrlHelper::Href(&self.url.borrow())
```

Comment:
> The href attribute’s getter and the toJSON() method, when invoked, must return the serialization of context object’s url. 

Per the spec, `href getter` and `toJSON` do the same thing to get result.

Also, I found the implementation in Gecko is calling the `href getter`.

https://hg.mozilla.org/mozilla-central/rev/619d60061378

Would you think that we should do `self.Href()` instead?

In my understanding, `UrlHelper::Href` has retrieved the same logic out of this module so maybe it's fine to use `UrlHelper::Href` in both `href getter` and `toJSON` ?

Reply:
Yeah, I'd just do `self.Href()`.

Reply:
Sure! Let me update! Thanks :D